release-deployment.md

```diff
@@ -127,22 +130,21 @@ There's nothing special about that. Each developer follows the above [Develop a
    * Description: Include a high-level list of things changed in this release.
    Click `Publish release`.
 
-1. Merge the `release-vX.Y.Z` into `develop`.
+1. Merge `master` into `develop`.
 
     ```
+    $ git checkout master
+    $ git pull
     $ git checkout develop
-    $ git merge release-vX.Y.Z
+    $ git merge master
     $ git push
     ```
 
-1. Finish off the tasks in the release checklist. Once everything is done, close
-   the release PR.
+1. Finish off the tasks in the release checklist.
```

Comment:
At this point the PR should already be merged and closed (reference earlier steps).